### PR TITLE
[DC-2382] visit_detail and note_nlp tables dropped by remove_extra_tables CR

### PR DESCRIPTION
**Pull request by nishanthpp93**
* [DC-2382] Use dynamic list of final tables

[DC-2382]: https://precisionmedicineinitiative.atlassian.net/browse/DC-2382?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: data_steward/cdr_cleaner/cleaning_rules/remove_extra_tables.py [hunk 1]
```diff
@@ -123,11 +55,22 @@ def __init__(self,
         this SQL, append them to the list of Jira Issues.
         DO NOT REMOVE ORIGINAL JIRA ISSUE NUMBERS!
         """
-        desc = 'Remove any tables that are not OMOP, OMOP extension, or Vocabulary tables.'
+        desc = 'Remove any tables that are not OMOP, OMOP extension, Vocabulary or AOU custom tables.'
+        # Use custom cdr_metadata instead of metadata
+        cdm_achilles_vocab_tables = list(
+            set(
+                cdm_schemas(include_achilles=True, include_vocabulary=True).
+                keys()) - {'metadata'}) + ['_cdr_metadata']
```
Comment (ksdkalluri) [inline data_steward/cdr_cleaner/cleaning_rules/remove_extra_tables.py:63]: you would need a metadata table as it is one of the tables added in CDM 5.3 https://ohdsi.github.io/CommonDataModel/cdm53.html#Metadata_Tables

**Review by ksdkalluri — CHANGES_REQUESTED** [review]

File: data_steward/cdr_cleaner/cleaning_rules/remove_extra_tables.py (hunk 1 above)
Comment (nishanthpp93) [reply data_steward/cdr_cleaner/cleaning_rules/remove_extra_tables.py:63]: I thought we were using `_cdr_metadata` instead of that. Are we using both?

**Comment by ksdkalluri** [comment]
@nishanthpp93 we are using the _cdr_metadata for info, but we should include all the tables in CDM although they are empty right.

**Review by ksdkalluri — APPROVED** [review]
changes look good!

**Pull request merged**